apps/web/src/components/NftSelector/NftSelector.tsx

```diff
@@ -350,20 +350,18 @@ function NftSelectorInner({ onSelectToken, headerText, preSelectedContract, even
         )}
       </StyledActionContainer>
 
-      {isLocked ? (
-        <NftSelectorLoadingView />
-      ) : (
-        <NftSelectorView
-          tokenRefs={tokensToDisplay}
-          selectedContractAddress={selectedContract?.address ?? null}
-          onSelectContract={handleSelectContract}
-          onSelectToken={onSelectToken}
-          eventFlow={eventFlow}
-          selectedNetworkView={network}
-          hasSearchKeyword={isSearching}
-          handleRefresh={handleRefresh}
-        />
-      )}
+      <NftSelectorTokens
```

Comment:
moved the loading state and selector view into a single component, so that I could return early with the announcement view in certain cases